File: frontend/src/main/java/com/sakamoto/frontend/data/dto/UpdateProjectCommand.java
package com.sakamoto.frontend.data.dto;

import com.sakamoto.frontend.data.model.Difficulty;
import com.sakamoto.frontend.data.model.Task;

import java.util.List;

public record UpdateProjectCommand(String id, String name, String description, List<Task> tasks, Difficulty difficulty) {
}

File: projects/src/main/kotlin/com/sakamoto/projects/dto/UpdateProjectCommand.kt
package com.sakamoto.projects.dto

import com.sakamoto.projects.model.Difficulty
import com.sakamoto.projects.model.Task

data class UpdateProjectCommand(
    val id: String,
    val name: String,
    val description: String,
    val tasks: List<Task>,
    val difficulty: Difficulty
)
